### PR TITLE
add title for kubernetes plugin

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@openshift/team-documentation @gabemontero ptal

File: using_images/other_images/jenkins.adoc [hunk 1]
```diff
@@ -632,5 +632,9 @@ monitors the API server of {product-title} for updates to `BuildConfigs` and
 projects (when a `BuildConfig` is created) or starts jobs in the resulting
 projects (when a `Build` is started).
 
+[[kubernetes-plug-in]]
+== Kubernetes Plug-in
+
+The Kubernetes plugin is used to run Jenkins slaves as pods on your cluster.
```
Comment (ahardin-rh) [inline using_images/other_images/jenkins.adoc]: plug-in

**Review by gabemontero — APPROVED** [review]

**Comment by ahardin-rh** [comment]
@bparees One nit from me. Is this targeting OCP 3.6?

**Comment by bparees** [comment]
@ahardin-rh nit fixed.  it's applicable to any version of the doc that had this new paragraph.

**Comment by ahardin-rh** [comment]
@bparees Cool. Looks like this was introduced via https://github.com/openshift/openshift-docs/pull/4588 and will be introduced as of OCP 3.6.

**Comment by ahardin-rh** [comment]
No revision history needed

**Pull request merged**